Code/Framework/AzCore/Platform/Common/UnixLike/AzCore/Debug/Trace_UnixLike.cpp

```diff
@@ -121,21 +123,16 @@ namespace AZ::Debug
         }
         Debug::Trace::Instance().RawOutput(nullptr, "==================================================================\n");
 
-        // Continue the overridden signal handlers by exiting with the matching exit code based on the signal
-        switch (signal)
-        {
-            case SIGSEGV:
-                _exit(139);
-                break;
-            case SIGKILL:
-                _exit(137);
-                break;
-            case SIGTRAP:
-                _exit(133);
-                break;
-            default:
-                break;
-        }
+        // Continue to exit the process when a signal is caught by this signal handler. 
+        // The exit code will conform to https://tldp.org/LDP/abs/html/exitcodes.html.
+        // - Signal based exit codes will be calculated based on 128+signal
+        // - Exit codes will not be greater than 255
+        //
+        // Any exit code that results in anything outside of this range will result in 255: Exit status out of range
+        constexpr int exitCodeSignalBase = 128;
+        constexpr int maxExitCode = 255;
+        int signalExitCode = exitCodeSignalBase + signal;
+        _exit(signalExitCode<maxExitCode?signalExitCode:maxExitCode);
```

Comment:
whitespace would be nice.
```suggestion
        _exit(signalExitCode < maxExitCode ? signalExitCode : maxExitCode);
```